ajax/libs/vue-functional-calendar/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "vue-functional-calendar",
+  "title": "Vue.js Functional Calendar",
+  "description": "Lightweight, high-performance calendar component based on Vue.js",
+  "keywords": [
+    "vue",
+    "calendar",
+    "component",
+    "vue-component",
+    "functional",
+    "date picker",
+    "date range",
+    "range",
+    "multiple calendars",
+    "multiple",
+    "modal calendar",
+    "modal",
+    "inline",
+    "date",
+    "change month",
+    "change year",
+    "picker",
+    "day",
+    "month",
+    "year",
+    "time",
+    "add"
+  ],
+  "homepage": "https://github.com/ManukMinasyan/vue-functional-calendar",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ManukMinasyan/vue-functional-calendar.git"
+  },
+  "license": "MIT",
+  "author": "Manuk Minasyan <manuk.minasyan1@gmail.com> (https://github.com/ManukMinasyan)",
+  "filename": "vue-functional-calendar.min.js",
+  "autoupdate": {
+    "source": "git",
+    "target": "git@github.com:ManukMinasyan/vue-functional-calendar.git",
+    "fileMap": [
+      {
+        "basePath": "dist/",
+        "files": [
+          "**/*"
+        ]
+      }
+    ]
+  }
```

Comment:
Your repo doesn't contain a `dist` dir: https://github.com/ManukMinasyan/vue-functional-calendar/tree/v2.7.7
What should we use for auto-update instead?